esp/services/esdl_svc_engine/esdl_store.hpp

```diff
@@ -28,16 +28,23 @@
  #define esdl_engine_decl DECL_IMPORT
 #endif
 
+interface IEsdlDefinitionInfo : public IInterface
+{
+    virtual IProperties* queryMetadata() = 0;
```

Comment:
changed both queryMetadata and queryServices to return const references, and renamed to getMetadata and getServices to avoid confusion with linked object interfaces

Reply:
changed to const